src/pages/Editor/index.tsx

```diff
@@ -102,6 +102,19 @@ const Editor = () => {
   }, [theme]);
 
   // Functions:
+  const transform = (text: string, id?: string) => {
+    return text.replace(/{{([\s\S]+?)}}/g, (_, $1) => {
```

Comment:
@saxenasahilas can you confirm if this regex handles attributes with underscores and numbers as well?

If not, try this:
`const regex = /\{\{([a-zA-Z0-9_\-]+)\}\}/g;` (L5 in src\utils\extractAttributes.ts)